package.json

```diff
@@ -4,18 +4,19 @@
   "description": "Facade over bunyan logger.",
   "main": "index.js",
   "scripts": {
-    "coverage-generate": "yarn istanbul -- cover _mocha -- --recursive",
-    "coverage-check": "istanbul check-coverage --config .istanbul.yml",
-    "coverage-upload": "cat ./coverage/lcov.info | coveralls",
-    "git-hook": "yarn lint && yarn coverage-generate && yarn coverage-check",
-    "istanbul": "NODE_ENV=test istanbul",
+    "coverage-upload-coveralls": "cat ./coverage/lcov.info | coveralls",
+    "git-hook": "yarn lint && yarn test",
     "lint": "eslint .",
     "lint-watch": "esw --watch .",
-    "prepush": "yarn git-hook",
-    "precommit": "yarn git-hook",
-    "test": "NODE_ENV=test mocha --recursive test",
-    "test-ci": "yarn git-hook && yarn coverage-upload",
-    "test-watch": "yarn test -- --watch --reporter min"
+    "test": "nyc mocha --exit --recursive test",
+    "test-ci": "nyc --reporter=lcov --reporter=cobertura mocha --exit --recursive test --reporter mocha-junit-reporter",
```

Comment:
I'd suggest that `test-with-coverage` (or something similar) is a name which closer matches the behaviour.

test-ci is a hangover from an older incarnation where it consisted of of lint + coverage generate + coverage upload but each of these are individual steps in github actions.

Reply:
Good point. Updated.